changelogs/fragments/63609_nmcli.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- nmcli - Use 'connection.autoconnect' instead of 'autoconnect' (https://github.com/ansible-collections/community.general/issues/459).
```

Comment:
```suggestion
- nmcli - use 'connection.autoconnect' instead of 'autoconnect' (https://github.com/ansible-collections/community.general/issues/459).
```